docs/pages/installation/installation-rocm.md

```diff
@@ -72,6 +72,8 @@ Finally, build Aphrodite:
 git clone https://github.com/PygmalionAI/aphrodite-engine.git
 cd aphrodite-engine
 
+chmod +x ./amdpatch.sh
+./amdpatch.sh 
```

Comment:
Can we add this to the Dockerfile as well?

Reply:
I may do it in a follow-up PR.

Reply:
Not sure, cause I don't use docker. Though probably yes